File: 10Proxy/Students/dzduniak/C#/DynamicProxy/DynamicProxy/Target.cs
namespace DynamicProxy
{
    public class Target : ISomeInterface
    {
        public string Method1()
        {
            return this.GetType().Name + ": Tu metoda nr 1!";
        }

        public string Method2()
        {
            return this.GetType().Name + ": Tu metoda nr 2!";
        }
    }
}

File: 10Proxy/Students/dzduniak/Java/DynamicProxy/src/Target.java
public class Target implements SomeInterface {
    @Override
    public String method1() {
        return this.getClass().getName() + ": Tu metoda nr 1!";
    }

    @Override
    public String method2() {
        return this.getClass().getName() + ": Tu metoda nr 2!";
    }
}
